nitro-protocol/contracts/InterestBearingApp.sol

```diff
@@ -0,0 +1,150 @@
+pragma solidity 0.8.17;
+pragma experimental ABIEncoderV2;
+
+import './interfaces/IForceMoveApp.sol';
+import './libraries/NitroUtils.sol';
+import {ExitFormat as Outcome} from '@statechannels/exit-format/contracts/ExitFormat.sol';
+
+// InterestBearingApp is a ForceMoveApp that allows a lender to earn interest
+// on a deposit. It functions as a ConsensusApp with the following additional rule:
+//   - the lender can unilaterally transition from state n to state n+1,
+//     forcing calculated interest into the lender's Outcome allocation
+contract InterestBearingApp is IForceMoveApp {
+    struct Funds {
+        address[] asset;
+        uint256[] amount;
+    }
+
+    struct InterestAppData {
+        // the per-block interest rate, expressed as the denominator of a fraction
```

Comment:
This might be particular to my brain but it might help to specify the numerator is `1`?

Reply:
:+1: 32047eb4